<launch>
<node name="perception" pkg="realsense_perception" type="realsense_perception_node" respawn="false" />
</launch>
